From 33e2e56aec5439bb9a0987b93affe2e119f62b7f Mon Sep 17 00:00:00 2001
From: Nick Hollinghurst <nick.hollinghurst@raspberrypi.com>
Date: Tue, 19 Nov 2024 16:11:32 +0000
Subject: [PATCH] drm: bridge: panel: Connector to allow interlaced modes

When initialized from panel_bridge_attach(), connector should
allow interlaced modes rather than invariably rejecting them,
so that other components can validate them.

Signed-off-by: Nick Hollinghurst <nick.hollinghurst@raspberrypi.com>
---
 drivers/gpu/drm/bridge/panel.c | 2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/gpu/drm/bridge/panel.c
+++ b/drivers/gpu/drm/bridge/panel.c
@@ -79,6 +79,8 @@ static int panel_bridge_attach(struct dr
 		return ret;
 	}
 
+	connector->interlace_allowed = true;
+
 	drm_panel_bridge_set_orientation(connector, bridge);
 
 	drm_connector_attach_encoder(&panel_bridge->connector,
